torch/csrc/jit/ir/alias_analysis.cpp

```diff
@@ -1333,6 +1333,16 @@ bool AliasDb::mayContainAlias(
       : memoryDAG_->mayContainAlias(a_elems, getElements(b));
 }
 
+bool AliasDb::mayContainAlias(Value* a, const at::ArrayRef<Value*> b) const {
+  if (!isMutableTypeInternal(a)) {
+    return false;
+  }
+  auto b_elems = getElements(b);
+  return b_elems.size() == 0
```

Comment:
is it faster to check this twice vs. delegating the check to MemooryDAG::mayContainAlias?